ntbs-integration-tests/Helpers/Utilities.cs

```diff
@@ -10,9 +11,16 @@ public static class Utilities
         public const int NOTIFIED_ID = 2;
         public const int DENOTIFIED_ID = 3;
         public const int NEW_ID = 1000;
+
+        public const int DENOTIFY_WITH_DESCRIPTION = 10;
+        public const int DENOTIFY_NO_DESCRIPTION = 11;
+
         public static void SeedDatabase(NtbsContext db)
         {
             db.Notification.AddRange(GetSeedingNotifications());
+
+            db.Notification.AddRange(DenotifyPageTests.GetSeedingNotifications());

```

Comment:
Any reason we don't just add these seeding notifications to the method in here?

Reply:
With these ones, not really, but I think it's a nicer pattern to have the non-shared notifications seeded from inside of the test suite.

There's currently a skipped test in DenotifyPageTests for validation that notification date is before denotification date (not yet implemented).
Seeding inside the testing class allows someone looking at the code to have full visibility of any specific notifications for that class without navigating to external files.

Reply:
Ok, fair enough that makes sense - should we maybe move GetSeedingNotifications from here into TestRunnerBase, to follow that pattern a bit more closely? As the ones here are relevant for all tests

Reply:
I think it would be preferable to leave it here, TestRunnerBase is already quite busy - and keeping it here allows greater visibility.

Had a brief chat with Jan, and he doesn't seem particularly fussed either way - I can shift it out if you have a strong preference - but I do think leaving the shared seeds in here makes sense.

Reply:
That's fine by me!